srcs/parsing/check_map_grid.c

```diff
@@ -3,10 +3,10 @@
 /*                                                        :::      ::::::::   */
 /*   check_map_grid.c                                   :+:      :+:    :+:   */
 /*                                                    +:+ +:+         +:+     */
-/*   By: aweaver <aweaver@student.42.fr>            +#+  +:+       +#+        */
+/*   By: elouisia <elouisia@student.42.fr>          +#+  +:+       +#+        */
 /*                                                +#+#+#+#+#+   +#+           */
 /*   Created: 2022/10/03 09:27:26 by aweaver           #+#    #+#             */
-/*   Updated: 2022/10/29 17:36:07 by aweaver          ###   ########.fr       */
+/*   Updated: 2022/10/29 19:42:38 by elouisia         ###   ########.fr       */
```

Comment:
I love Reach but I also hate him for this header updating when nothing changed in the file (though to be fair this is us saving files that do not need to be saved).